### PR TITLE
Update title

### PR DESCRIPTION
**Pull request by JonPHardy57**
Updated the title to "Polling Tentacles over Standard HTTPS Port"

### CODE REVIEW

**Comment by michaelnoonan** [comment]
Not quite sure why I've been requested as a reviewer here. :)

**Comment by JonPHardy57** [comment]
Hi Mike,
You and Jim were the assigned reviewers by Github, I was a little surprised
as well. I thought that this was probably the arrangement to change from
Mark Harrison who was caretaking it (as I understood it at the time). So,
who do you think should be the reviewer and can I redirect it?
Alternatively this is a very simple change as requested by Colin, so to get
this moving can you do it? And  if Github is incorrectly configured, then
the reviewers for docs can be sorted out in parallel.

Many Thanks
Jon

Jon Hardy
Snr Product Manager| Octopus Deploy | Melbourne, Australia
[image: Octopus Deploy] <https://octopus.com/>

**Comment by michaelnoonan** [comment]
Gotcha. @harrisonmeister , I get the sense this pull request, as you've suggested in your comments, isn't quite right to achieve such a simple goal. Given your experience with the docs, @harrisonmeister are you willing and able to make the change @JonPHardy57 wants to achieve?

File: docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Polling Tentacles over Standard HTTPS Port
```
[the remark this answers is not in the document]
Comment (JonPHardy57) [reply docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443.md:2]: Hi Mark, I thought that I had just changed the title on the page. So, I am a bit puzzled. I Bing chat search is talking about editing the yaml in the config file. Not confident I want to do this and quite possibly make matters worse. So any pointers would be greatly appreciated. Thanks.

**Comment by harrisonmeister** [comment]
> Gotcha. @harrisonmeister , I get the sense this pull request, as you've suggested in your comments, isn't quite right to achieve such a simple goal. Given your experience with the docs, @harrisonmeister are you willing and able to make the change @JonPHardy57 wants to achieve?

Yup I can get this one pushed through in my day tomorrow @michaelnoonan

**Comment by michaelnoonan** [comment]
Much appreciated. Thank you.

**Comment by JonPHardy57** [comment]
Thank you both, it is appreciated.

**Review by harrisonmeister — APPROVED** [review]

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM!

File: docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (harrisonmeister) [reply docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443.md]: No worries @JonPHardy57 - I've fixed up the changes

The tl;dr of what happened is that when you created the branch initially, this was done from the `master` branch. Since the master branch has been around, the docs have now moved to the `main` branch, which includes a slightly different page structure.

When you made your change and then merged main into your `Update-Title` branch - you got both pages, your "renamed" title one, and the existing one (that had moved location in the filesystem). You probably had some merge conflicts to resolve with the initial master -> `Update-Title` branch and kept your file with the new name as after all that's what you wanted to change!

So all I did, was delete your "new" page and apply the title change to the existing page.

Moving forward there its very unlikely this could happen again as @steve-fenton-octopus has deleted the `master` branch just now - I'd recommend you delete the master branch from your remote fork/local git repo too!

Hope that helps explain! :smile:

**Comment by harrisonmeister** [comment]
> Not quite sure why I've been requested as a reviewer here. :)

Just to close the loop on this @michaelnoonan - I _think_ the reason you were requested as a reviewer was presumably due to the branch originating from `master` which used to have a codeowners file - I thought that was only for the credits file, but the fact you and Jim were requested gives me clues to suspect this was the reason

All sorted now, thanks for looping me in!
